src/components/ContentList/styles.js

```diff
@@ -0,0 +1,72 @@
+import { StyleSheet } from 'react-native';
+import * as colors from 'kitsu/constants/colors';
+
+export const styles = StyleSheet.create({
+  contentListContainer: {
+    paddingHorizontal: 10,
+    paddingVertical: 10,
```

Comment:
what about `padding: 10`

Reply:
`padding` property support has landed in React Native? Sweet! I wasn't aware of that.